roles/kube_nginx_ingress/files/nginx_ingress_values.yml

```diff
@@ -0,0 +1,6 @@
+controller:
+    hostNetwork: True
+    kind: 'DaemonSet'
+
```

Comment:
Even though we currently don't use them yet, setting `controller.stats.enabled=true` may be useful.